README.md

```diff
@@ -39,9 +39,22 @@ https://hacks.mozilla.org/2019/04/teaching-machines-to-triage-firefox-bugs/
 - **uplift** - The aim of this classifier is to detect bugs for which uplift should be approved and bugs for which uplift should not be approved.
 
 
-## Setup
+## Setup and Prerequisites
 
-Run `pip install -r requirements.txt` and `pip install -r test-requirements.txt`. Depending on the parts of bugbug you want to run, you might need to install dependencies from other requirement files (find them with `find . -name "*requirements*"`).
+libgit2 (needs [v1.0.0](https://github.com/libgit2/libgit2/releases/tag/v1.0.0), only in [experimental on Debian](https://wiki.debian.org/DebianExperimental))
```

Comment:
I'm using 0.28.5+dfsg.1-1 from Debian testing, did that not work for you?

Reply:
No, that failed at `pip3 install`, where one package does native compilation (probably pygit2), with an error message stating that v0.99 or v1.0.0 was required.

Reply:
Strange, maybe for some reason pip is building it from source for you, and installing the wheel for me.